### ISSUE TITLE
Update Published Version

### ISSUE THREAD

**Issue by timkendall** [DockYard/ember-async-button#84]
It looks like the build with support for closure actions never got a new version and corresponding npm publish.

**Comment by blimmer** [comment]
As a workaround you can do this:
```
ember install https://github.com/DockYard/ember-async-button.git#7ecc1418c6ac0603eafdf2d
8ceca3eeb54f5f39f
```

**Comment by blimmer** [comment]
ping @bcardarella - can you please publish an update to npm?

**Comment by bcardarella** [comment]
I just published 1.0.0 of the library.

The future of this library will be maintenance. We are exploring replacing this library with `ember-concurrency`.
